src/test/java/net/masterthought/cucumber/generators/FeatureReportPageTest.java

```diff
@@ -49,4 +52,25 @@ public void prepareReport_AddsCustomProperties() {
         assertThat(context.get("parallel")).isEqualTo(configuration.isParallelTesting());
         assertThat(context.get("feature")).isEqualTo(feature);
     }
+
+    @Test
+    public void getMimeType_OnEmbeddingFromV2CucumberReportFile_SupportsScreenshots() {
+        testScreenshotsAreSupported(CUCUMBER_V2_WITH_SCREENSHOT_JSON);
+    }
+
+    @Test
+    public void getMimeType_OnEmbeddingFromV3CucumberReportFile_SupportsScreenshots() {
+        testScreenshotsAreSupported(CUCUMBER_V3_WITH_SCREENSHOT_JSON);
+    }
+
+    private void testScreenshotsAreSupported(String reportFile) {
```

Comment:
can you use above test method naming convention?

Reply:
In order to have `//given` `//when` `//then` comments as above and to have asserts to appease codacy I've removed this method.